docs_2.0/manual_EN/5.deployment-and-installation/1.resource-preparations.md

```diff
@@ -0,0 +1,181 @@
+| gettext          | Any mainstream version | -                                                         |
+| curl             | Any mainstream version | -                                                         |
+| redhat-lsb-core  | Any mainstream version | -                                                         |
+| libstdc++-static | Any mainstream version | Only needed in CentOS 8+, RedHat 8+, and Fedora systems.  |
+| libasan          | Any mainstream version | Only needed in CentOS 8+, RedHat 8+, and Fedora systems.  |
+
+## Requirements and suggestions for installing Nebula Graph in test environments
+
+### Hardware requirements for test environments
+
+| Item               | Requirement |
+| ------------------ | ----------- |
+| CPU architecture   | x86_64      |
+| Number of CPU core | 4           |
+| Memory             | 8 GB        |
+| Storage            | 100 GB, SSD |
+
+### Supported operating systems for test environments
+
+For now, we can only install Nebula Graph in the Linux system. To install Nebula Graph in a test environment, we recommend that you use any Linux system with kernel version 2.6.32 or above.
+
+You can adjust some of the kernel parameters to better accommodate the need for running Nebula Graph. For more information, see [Optimize Linux kernel configuration](TODO) [TODO].
+
+### Suggested service architecture for test environments
+
+| Process                                   | Suggested number |
+| ----------------------------------------- | ---------------- |
+| metad (the metadata service process)      | 1                |
+| storaged (the storage service process)    | 1 or more        |
+| graphd (the query engine service process) | 1 or more        |
+
+For example, on a cluster of 3 machines (named as A, B, and C), you can deploy Nebula Graph as follows:
+
+| Machine name | Number of metad | Number of storaged | Number of graphd |
+| ------------ | --------------- | ------------------ | ---------------- |
+| A            | 1               | 1                  | 1                |
+| B            | None            | 1                  | 1                |
+| C            | None            | 1                  | 1                |
+
+## Requirements and suggestions for installing Nebula Graph in production environments
+
+### Hardware requirements for production environments
+
+| Item               | Requirement          |
+| ------------------ | -------------------- |
+| CPU architecture   | x86_64               |
+| Number of CPU core | 48                   |
+| Memory             | 96 GB                |
+| Storage            | 2 * 900 GB, NVMe SSD |
+
+### Supported operating systems for production environments
+
+For now, we can only install Nebula Graph in the Linux system. To install Nebula Graph in a production environment, we recommend that you use any Linux system with kernel version 3.9 or above.
```

Comment:
why the kernel version is different from the test version?

Reply:
Thank you. I misunderstood it. They should be the same. Fixed.